### ISSUE TITLE
Improvement - Board rescan button

### ISSUE THREAD

**Issue by MobiFlight-Admin** [MobiFlight/MobiFlight-Connector#354]
## Problem summary

There is no way to rescan the boards connected to the PC from inside MF. 

## MF Version

Latest

## Steps to reproduce

* Start MF
* Connect new board to PC
* MF does not do anything 

## Expected result

It would be nice to have a button to rescan for any boards newly connected to the PC. 

## Actual result

There is no way to rescan, so only solution is to exit and restart MF. 

## Further details

‌

[Issue created by jaime709: 2021-05-07]
[Last updated on bitbucket: 2021-06-08]

[Comment created by tigert: 2021-06-08]
This could be in the dropdown menu that lists your boards on the status bar:

```
| Modules (v) | Sim status | SimConnect Status | 
        |    `-------------------------.
        | Autopilot / SN-11f-10b       |
        | Switch panel / SN-A5d-32c    |
        +------------------------------+
        | Re-scan connected modules    |
        `------------------------------'
```

‌

**Comment by storeilly** [comment]
I'd like to add my vote for this feature please

**Comment by MobiFlight-Admin** [comment]
closed in favor of #1091 - there won't be the need for a button once #1091 is implemented
